lib/jekyll-github-metadata/site_github_munger.rb

```diff
@@ -42,10 +42,13 @@ def drop
 
       # Set `site.url` and `site.baseurl` if unset.
       def add_url_and_baseurl_fallbacks!
-        site.config["url"] ||= Value.new("url", proc { |_c, r| r.url_without_path })
+        Jekyll::GitHubMetadata.log :debug, "Adding URL and base URL fallbacks..."
```

Comment:
This describes the whole function, so some whitespace is useful, so indicate it applies to the section and not just the line or two directly after it.
```suggestion
        Jekyll::GitHubMetadata.log :debug, "Adding URL and base URL fallbacks..."

```